### PR TITLE
fix: Use 'utf-8' -> 'latin-1' -> 'utf-8' to remove non-latin characters

### PR DESCRIPTION
**Pull request by dheshanm**
Addresses #114

### CODE REVIEW

File: nda-transform/encode_to_latin.py [hunk 1]
```diff
@@ -51,10 +51,16 @@ def main():
...
             temp_path = Path(temp_file.name)
-            shutil.copy(temp_path, filename)
+
+            print(f"Saving 'utf-8' -> 'latin-1' -> 'utf-8' encoded file to {filename}")
+            with tempfile.NamedTemporaryFile("w", encoding="utf-8") as temp_file_utf8:
+                with open(temp_path, "r", encoding="latin-1") as temp_file_latin1:
+                    temp_file_utf8.write(temp_file_latin1.read())
+
+                temp_file_utf8.flush()
+                shutil.copy(temp_file_utf8.name, filename)
```
Comment (tashrifbillah) [inline nda-transform/encode_to_latin.py:63]: So you are just saving to a temporary file before writing in the official location?

Comment (dheshanm) [reply nda-transform/encode_to_latin.py:63]: Yes. I use `tempfiles` to avoid having to delete the intermediary files I generate.

**Comment by tashrifbillah** [comment]
Hi Dheshan, I think this work is still not right:

image

The `???` characters will be rejected by NDA validation tool. Or did we agree on letting the rejection happen?

**Comment by tashrifbillah** [comment]
We tried a lot. It is just so hard to do.

**Comment by tashrifbillah** [comment]
Due to conflict, merged via https://github.com/AMP-SCZ/utility/commit/bde2031090b8bbe75af59c9747392fbbad3dd6f4

**Pull request closed without merging**